pagopa/io-app
# feat: [IOCOM-429,IOCOM-433] PN Multiple Payments: actions, reducer and sagas

## Short description
This PR adds actions, reducer and sagas to handle multiple payment updates (PN section only).

## List of changes proposed in this pull request
- Two new actions: an async one to request/cancel single updates and one to stop all enqueued ones;
- A new reducer that tracks multiple payments' statuses on multiple messages
- A new saga that handles configurable buffered updates
- `paymentVerificationRequestHandler` has been refactored in order to share common code flow with the PN payment update procedure

## How to test
Since the UI is not there yet, you can either check the code flow or execute the flow by manually launching the desired actions.
